docs/conf.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3.7
+#!/usr/bin/env python3.9
```

Comment:
```suggestion
#!/usr/bin/env python3
```

Why not just do this so we don't have to worry about future maintenance of it?

Reply:
Oh...good thinking!